SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialRPCService.cpp

```diff
@@ -69,7 +83,7 @@ EPushRPCResult SpatialRPCService::PushRPCInternal(Worker_EntityId EntityId, ERPC
 			return EPushRPCResult::NoRingBufferAuthority;
 		}
 
-		EndpointObject = Schema_GetComponentUpdateFields(GetOrCreateComponentUpdate(EntityComponent));
+		EndpointObject = Schema_GetComponentUpdateFields(GetOrCreateComponentUpdate(EntityComponent, Payload.SpanId.IsSet() ? &Payload.SpanId.GetValue() : nullptr));
```

Comment:
Is passing the span ID to `GetOrCreateComponentUpdate` a leftover from when Danny was thinking that we might have to put the span ID in the RPC component update itself? Shouldn't be necessary.

Reply:
This is actually how we get the Span Ids for sening RPCs at the point of sending them using the worker API.